### ISSUE TITLE
How much memory is consumed during the training.

### ISSUE THREAD

**Issue by chenkang455** [codingrex/TimeRewind#1]
Hi @codingrex , thank you for your great work. You mentioned that testing requires about 10GB memory of GPU. May I ask how much memory is approximately consumed during your training?

**Comment by codingrex** [comment]
Hi @chenkang455, thanks very much for your interest and question. The training with per GPU batch size 1 cost around 33GB GPU memory per GPU. Our training used distributed training on multiple GPUs and used gradient accumulation to achieve effective training batch size > 1.

**Comment by chenkang455** [comment]
> Hi @chenkang455, thanks very much for your interest and question. The training with per GPU batch size 1 cost around 33GB GPU memory per GPU. Our training used distributed training on multiple GPUs and used gradient accumulation to achieve effective training batch size > 1.

Got it, thanks for your reply!
